ffi/src/dpapi/api.rs

```diff
@@ -0,0 +1,42 @@
+#[cfg(not(any(test, miri)))]
+mod inner {
+    pub use dpapi::{n_crypt_protect_secret, n_crypt_unprotect_secret};
+}
+
+#[cfg(any(test, miri))]
+mod inner {
+    /// We have FFI wrappers for DPAPI functions from the [dpapi] crate and we want to test them.
+    /// The DPAPI implementation is complex and makes calls to the RPC and KDC servers.
+    /// Implementing a mock for KDF and RPC servers is too hard and unreasonable. So, we wrote a simple
+    /// high-level mock of [n_crypt_protect_secret] and [n_crypt_unprotect_secret] functions.
+    ///
+    /// **Note**: The goal is to test FFI functions, not the DPAPI implementation correctness.
+    /// The FFI tests should not care about returned data correctness but rather check
+    /// for memory corruptions and memory leaks.
```

Comment:
issue: You probably want the docstring to apply to the item above (the module)
```suggestion
    //! We have FFI wrappers for DPAPI functions from the [dpapi] crate and we want to test them.
    //! The DPAPI implementation is complex and makes calls to the RPC and KDC servers.
    //! Implementing a mock for KDF and RPC servers is too hard and unreasonable. So, we wrote a simple
    //! high-level mock of [n_crypt_protect_secret] and [n_crypt_unprotect_secret] functions.
    //!
    //! **Note**: The goal is to test FFI functions, not the DPAPI implementation correctness.
    //! The FFI tests should not care about returned data correctness but rather check
    //! for memory corruptions and memory leaks.

```

Reply:
Oh, yes. thanks!
done